applications/Unity.GrantManager/modules/Unity.Payments/src/Unity.Payments.Application.Contracts/Integrations/Cas/InvoiceLineDetail.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Unity.Payments.Integrations.Cas
+{
+    public class InvoiceLineDetail : IValidatableObject
+    {
+        [Required]
+        public int invoiceLineNumber { get; set; }
+
+        [Required]
+        [MaxLength(4, ErrorMessage = "invoiceLineType must be 'Item'")]
+        public string invoiceLineType { get; set; } = "Item";

```

Comment:
Pascal case and json property name for serialization name, as as above pls

Reply:
Done, CAS expects these as lower case and throws an exception for the cap case. Had to add decorator property to resolve this and move away from NewtonSoft JSON serialization. The code base I coppied this from must have had the same issue and made them lower case for conveniance of conversion.